cpp/oneapi/dal/algo/knn/common.hpp

```diff
@@ -209,6 +244,14 @@ class descriptor : public detail::descriptor_base<Task> {
         set_neighbor_count(neighbor_count);
     }
 
+    /// Creates a new instance of the class with the given :literal:`neighbor_count`
+    /// property value.
+    /// Used with :expr:`task::search` only.
+    template <typename T = Task, typename = detail::enable_if_search_t<T>>
+    explicit descriptor(std::int64_t neighbor_count) : base_t() {
```

Comment:
```suggestion
    explicit descriptor(std::int64_t neighbor_count) {
```

Reply:
Why shouldn't we initialize base explicitly?

Reply:
No + we never do it

Reply:
Fixed